app/scripts/components/reconciliation-reports/report-table.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import SortableTable from '../table/sortable';
+
+class ReportTable extends React.Component {
+  render () {
+    const {
+      data,
+      title,
+      tableHeader,
+      tableRow,
+      tableProps
+    } = this.props;
+
+    if (!data || !data.length) {
+      return null;
+    }
+
+    return (
+      <div className='page__section--small'>
```

Comment:
Is it possible to add `Hide` and `Show` capability for listing files/granules?   There could be long list of discrepancy.